packages/grafana-icons/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@grafana/icons",
```

Comment:
@staton-hyse11 this is the name under which the package will be published. Which ideas for the name did you have in mind?

Reply:
@Clarity-89 "@grafana/saga-icons" makes sense to me here. it keeps the pattern of "@grafana" but notes these are part of the design system.